README.md

```diff
@@ -83,6 +83,11 @@ triggers related tasks in subprojects.
 
 Import the [settings.gradle.kts](settings.gradle.kts) file into your IDE.
 
+To enable Android sample add `local.properties` file to project root folder with following content:
+```properties
+sdk.dir=<android-sdk-location>
+```
```

Comment:
I think this is very close to being a solution, however, you haven't needed to do this in the past. So what is the android gradle plugin looking for/finding that isn't this key?

Can you do the same sort of search that the plugin does?

Reply:
Plugin is looking for either `ANDROID_HOME` variable or `sdk.dir` in `local.properties`. Nothing fancy here. I will also add a check for variable.